benchmarks/tpcc/sysbench-tpcc.sh

```diff
@@ -1622,6 +1680,11 @@ for function in "${functions[@]}"; do
 
     # Check if an error occurred
     if [ $return_value -ne 0 ]; then
+        get_container_logs
+        stop_containers
+        remove_containers
+        delete_container_data
+
         error_msg "An error occurred in '$function'. Exiting."
```

Comment:
Please add
error_msg "... See container logs at '${OUTPUT_PATH}/${OUTPUT_PREFIX}mysql.*.log' for details"

Reply:
1497db9afb22